wiki/Performance_Troubleshooting/vi.md

```diff
@@ -0,0 +1,105 @@
+
+- Bấm Ctrl+F11 để xem biểu đồ biểu thị thời gian khung hình. Kiểm tra xem liệu có xuất hiện nhiều đốm/vạch trắng trên biểu đồ khi việc này xảy ra hay không (nó có nghĩa là một ứng dụng khác, hoặc driver trên hệ thống của bạn đang dùng quá nhiều CPU time, làm trễ tín hiệu vào).
+
+### FPS thấp
+
+Khi chơi, FPS thấp quá làm cho các vật thể xuất hiện trên màn hình trông giật và di chuyển kỳ quặc. Điều này cũng có thể khiến tăng độ trễ của tín hiệu vào. Khi chẩn đoán hoặc báo cáo, bạn hãy thử:
+
+- Kiểm tra xem liệu nó vẫn xảy ra sau khi khởi động lại máy.
+- - Thử đóng các ứng dụng mà hiện overlay trên game (như Discord, Steam, NVIDIA Shadowplay, v.v.)
+- Đảm bảo bạn không bật đồng bộ dọc (Vertical Sync) ở bất cứ driver của card đồ hoạ nào (nên để tắt "Off" hoặc tuỳ thiết lập của ứng dụng "Application-controlled")
+- Thử thay đổi Giới hạn khung hình xem liệu nó có cải thiện tình trạng hay không.
+- Hãy đảm bảo bạn chạy osu! ở chế độ Toàn màn hình (không phải Không viền). Điều này sẽ luôn cải thiện hiệu năng.
+- Đảm bảo bạn đang bật [Game mode](https://quantrimang.com/huong-dan-kich-hoat-che-do-game-mode-tren-windows-10-130041). Nó cho phép Windows cung cấp nhân (CPU/GPU) và tài nguyên dành riêng cho osu!.
+- Mở Task manager, chuyển sang tab Details và sắp xếp danh sách theo thứ tự tiêu tốn CPU. Kiểm tra xem mỗi khi xảy ra đứng khung hình, có ứng dụng nào tiêu tốn CPU nhiều không, như virus chạy dưới nền, hoặc trình quét diệt virus. Bạn có thể chạy osu! ở chế độ cửa sổ, hoặc để Task Manager ở màn hình thứ hai để chẩn đoán hiện tượng khi đang chơi hoặc chạy beatmap với mod Tự động.
+
+## Giới hạn khung hình
+
+osu! cho bạn nhiều lựa chọn giới hạn khung hình. Chúng đều có ưu và nhược điểm riêng. Chú ý rằng ở bất cứ thiết đặt nào, osu! sẽ giới hạn khung hình ở menu để hạn chế việc sử dụng CPU.
+
+### VSync (Đồng bộ dọc)
+
+Mức giới hạn này phụ thuộc vào cơ chế đồng bộ dọc của driver. Nó đảm bảo không có hiện tượng xé hình, nhưng sẽ làm tăng độ trễ khoảng 1-2 khung hình vì cần phải buffer khung hình trước khi hiển thị.
+
+Không khuyến nghị cho màn hình 60Hz. Nhưng nó có thể hữu ích hơn ở màn hình trên 120Hz, nhưng với màn hình có GSync hay FreeSync, thì việc dùng VSync để tránh xé hình không còn cần thiết nữa.
+
+### Power Saving (Tiết kiệm điện)
+
+Mức giới hạn này nhắm tới việc tiết kiệm điện trong khi vẫn đáp ứng được trải nghiệm game ổn định. Game sẽ chạy với fps bằng 2 lần tần số quét của màn hình và cố gắng giữ ổn định tần suất khung hình.
+
+Khuyến nghị cho máy tính cổ hoặc laptop cần tiết kiệm pin.
+
+### Optimal (Tối ưu)
+
+Mức giới hạn này giữ tần suất khung hình bằng 8 lần tần số quét màn hình, tối đa là 960fps. Đây là mức giới hạn được khuyến nghị cho người dùng không muốn đốt CPU/GPU quá mức, nhưng vẫn đảm bảo hiệu năng đủ cao dẫn đến độ trễ tín hiệu vào thấp.
```

Comment:
```suggestion
Mức giới hạn này giữ tần suất khung hình bằng 8 lần tần số quét màn hình, tối đa là 960 FPS Đây là mức giới hạn được khuyến nghị cho người dùng không muốn đốt CPU/GPU quá mức, nhưng vẫn đảm bảo hiệu năng đủ cao dẫn đến độ trễ tín hiệu vào thấp.
```

Reply:
You deleted the period. But I've changed to capital text with a period. Thanks.